### DISCUSSION TITLE
Storage no longer supports range requests?

### DISCUSSION THREAD

**Discussion by mirkonasato** [supabase/supabase#4115] [Questions] [answered]
I tried Supabase Storage a few days ago to see if it supports byte range requests when fetching videos. They didn't work locally when starting Docker containers with the Supabase CLI, but I'm pretty sure they did work fine in the hosted production environment.

Today I ran the same test app again and ranges no longer work in production either? When fetching a URL generated with `createSignedUrl` the response does not include the `Accept-Ranges` header.

Form a quick search that feature should definitely be supported:
[Supabase Storage now in Beta: Streaming Media](https://supabase.com/blog/2021/07/27/storage-beta#streaming-media)
[Support byte range requests #20](https://github.com/supabase/storage-api/issues/20)

Anyone can confirm if it currently works for you?

**Comment by mirkonasato** [comment, marked as answer]
Well, today it's working fine again. I assume it was a temporary issue.

**Reply by e0** [reply]
I contacted the support about it yesterday (also linking to this discussion) and they got back to me and fixed it. The support response time was great. Thanks for creating the issue here @mirkonasato, it helped me confirm that the problem wasn't just on my side.
